docs/documentation/guides/event-filter.mdx

```diff
@@ -0,0 +1,503 @@
+---
+title: "Event Filters"
+description: "A guide to using event filters to create more precise and tailored job triggers."
+---
+
+## What are Event Filters?
+
+Event filters provide a powerful way to define specific conditions that determine when a job should be triggered based on incoming events. Filters allow you to narrow down event processing to only those events that match certain criteria, ensuring that your jobs are executed in a targeted manner.
+
+we will cover the various types of event filters and how they can be used to create more precise triggers.
+
+### Event Filter Types
+
+#### 1. String Filters
```

Comment:
Remove the numbers, should just be `### String Filters`.

Also no need for the `### Event Filter Types` header